include/GafferScene/VisibleSet.h

```diff
@@ -63,10 +63,31 @@ struct GAFFERSCENE_API VisibleSet
 	PathMatcher inclusions;
 	PathMatcher exclusions;
 
-	/// Returns the result of a match made against the VisibleSet.
-	/// ExactMatch : The location should be rendered.
-	/// DescendantMatch : Some (but not necessarily all) descendants of the location should be rendered (but this location shouldn't be unless ExactMatch is also set).
-	PathMatcher::Result match( const std::vector<InternedString> &path, const size_t minimumExpansionDepth = 0 ) const;
+	struct Result
+	{
+		enum DrawMode
+		{
+			None = 0,
+			Visible = 1,
+			ExcludedBounds = 2,
+		};
+
+		Result( DrawMode drawMode = DrawMode::None, bool descendantsVisible = false )
+			: drawMode( drawMode ), descendantsVisible( descendantsVisible )
+		{
+		}
+
+		bool operator == ( const Result &rhs ) const
+		{
+			return drawMode == rhs.drawMode && descendantsVisible == rhs.descendantsVisible;
+		}
+
+		DrawMode drawMode;
+		bool descendantsVisible;
+	};
+
+	/// Returns the Result of a match made against the VisibleSet.
+	Result match( const std::vector<InternedString> &path, const size_t minimumExpansionDepth = 0 ) const;
```

Comment:
We talked about it maybe making sense to rename some of this stuff now it's moved further from the PathMatcher interface. For the sake of argument, what do you think to renaming `Result` to `Visibility` and `match()` to `visibility()`?

Reply:
Both changes sound good to me, renamed in a6137b3, a63c1b2 & 379aa20. I'll adjust the commit messages once these get squashed down.